assets/javascripts/discourse/components/chat-channel-settings-btn.js

```diff
@@ -44,4 +58,18 @@ export default Component.extend({
   archiveChannel() {
     showModal("chat-channel-archive-modal").set("chatChannel", this.channel);
   },
+
+  closeChannel() {
+    showModal("chat-channel-toggle-open-modal").set(
+      "chatChannel",
+      this.channel
+    );
+  },
+
+  openChannel() {
+    showModal("chat-channel-toggle-open-modal").set(
+      "chatChannel",
+      this.channel
+    );
+  },
```

Comment:
It looks like both `closeChannel` and `openChannel` actions are doing the same thing. Is it possible to merge them into one action `toggleChannel` instead?

Reply:
Oh yeah makes sense...I probably made these at the start then forgot to update :sweat_smile: